tasks/managed/collect-pyxis-params/README.md

```diff
@@ -6,9 +6,19 @@ for downstream tasks to use.
 
 ## Parameters
 
-| Name     | Description                                                                                                            | Optional | Default value |
-|----------|------------------------------------------------------------------------------------------------------------------------|----------|---------------|
-| dataPath | Path to the merged data JSON file generated by collect-data task and containing the pyxis configuration options to use | No       | -             |
+| Name                    | Description                                                                                                                | Optional | Default value                                             |
+|-------------------------|----------------------------------------------------------------------------------------------------------------------------|----------|-----------------------------------------------------------|
+| dataPath                | Path to the merged data JSON file generated by collect-data task and containing the pyxis configuration options to use     | No       | -                                                         |
+| ociStorage              | The OCI repository where the Trusted Artifacts are stored                                                                  | Yes      | empty                                                     |
+| ociArtifactExpiresAfter | Expiration date for the trusted artifacts created in the OCI repository. An empty string means the artifacts do not expire | Yes      | 1d                                                        |
```

Comment:
missing trustedArtifactsDebug and orasOptions

Reply:
fixed